### PR TITLE
azuredatastudio-insiders: download nightly instead of stable

### PR DESCRIPTION
**Pull request by Bassetts**
URL was pointing to the stable version of Azure Data Studio. Updated to correctly download the Insiders edition and updated file and shortcut names as appropriate.

### CODE REVIEW

File: bucket/azuredatastudio-insiders.json [hunk 1]
```diff
@@ -1,22 +1,20 @@
 {
     "homepage": "https://docs.microsoft.com/en-us/sql/azure-data-studio",
-    "version": "1.12.2",
+    "version": "nightly",
     "description": "A cross-platform database tool for data professionals using the Microsoft family of on-premises and cloud data platforms.",
     "license": "MIT",
-    "bin": "bin\\azuredatastudio.cmd",
+    "bin": [
```
Comment (Ash258) [inline bucket/azuredatastudio-insiders.json]: Why did you change this?

```suggestion
    "bin": "bin\\azuredatastudio.cmd",
```

Comment (Ash258) [reply bucket/azuredatastudio-insiders.json]: Then just change the file name and do not add additional rename.

Edited original suggstion

File: bucket/azuredatastudio-insiders.json [hunk 2]
```diff
@@ -1,22 +1,15 @@
...
-    "url": "https://github.com/Microsoft/azuredatastudio/releases/download/1.12.2/azuredatastudio-windows-1.12.2.zip",
-    "hash": "89c0b9ef1223f6010802556524ee3b5042e7ebfa5e5513412bc35cbeac707b7f",
-    "extract_dir": "azuredatastudio-windows",
-    "checkver": {
-        "github": "https://github.com/Microsoft/azuredatastudio"
-    },
-    "autoupdate": {
-        "url": "https://github.com/Microsoft/azuredatastudio/releases/download/$version/azuredatastudio-windows-$version.zip"
-    }
+    "url": "https://azuredatastudio-update.azurewebsites.net/latest/win32-x64-archive/insider#/dl.7z",
```
Comment (Ash258) [inline bucket/azuredatastudio-insiders.json]: Move `url` and `extract_dir` after `license` property

Comment (Bassetts) [reply bucket/azuredatastudio-insiders.json]: Both `url` and `extract_dir` are already after the `license` property

Comment (Ash258) [reply bucket/azuredatastudio-insiders.json]: Not directly. Let me rephrase sentence to be more clear.

Put `url` and `hash` directly after `license` property and before `bin` property.
